eng/pipelines/templates/stages/archetype-java-release.yml

```diff
@@ -188,7 +200,11 @@ stages:
                 runOnce:
                   deploy:
                     steps:
-                      - checkout: self
+                      - template: /eng/common/pipelines/templates/steps/sparse-checkout.yml
+                        parameters:
+                          paths:
+                            - CODEOWNERS
```

Comment:
What this? 

If we need the CODEOWNERS files to get the pr owners? I will say we plan to remove this from this release pipeline now that we aren't submitting PRs for the docs PRs but we just haven't cleaned this up yet.

Reply:
Yeah there are 2 things one is the clean-up of the need for having to parse CODEOWNERS file under .github/CODEOWNERS. However my other question is why are you adding CODEOWNERS at the root as a path as well?

Reply:
This was my mistake. Fixed to only include `.github/CODEOWNERS`.